core/src/test/java/nl/nn/adapterframework/receivers/ReceiverTest.java

```diff
@@ -228,19 +224,10 @@ public ITransactionalStorage<Serializable> setupErrorStorage() {
 
 	public static Stream<Arguments> transactionManagers() {
 		return Stream.of(
-			Arguments.of(supplier(ReceiverTest::buildNarayanaTransactionManagerConfiguration)),
-			Arguments.of(supplier(ReceiverTest::buildBtmTransactionManagerConfiguration))
+			Arguments.of(supplier(ReceiverTest::buildNarayanaTransactionManagerConfiguration))
```

Comment:
Since there's still a chance that we might add new transaction manager implementations in future I think we should keep these tests parameterised.